translate_demo.py

```diff
@@ -50,6 +50,7 @@
 parser.add_argument('--upscale-ratio', default=None, type=int, choices=[1, 2, 4, 8, 16, 32], help='waifu2x image upscale ratio')
 parser.add_argument('--manga2eng', action='store_true', help='Render english text translated from manga with some typesetting')
 parser.add_argument('--eng-font', default='fonts/comic shanns 2.ttf', type=str, help='Path to font used by manga2eng mode')
+parser.add_argument('--font-filename', default='', type=str, help='Path to font used by text_render.prepare_renderer')

```

Comment:
1. I think `--font-path` might be better.
2. Please merge `--eng-font` and the new argument (Set the default value of eng-font in `dispatch_eng_render`
3. Can you also simplify the help? It doesnt need to include any specific code.

Reply:
changed to `--font-path`
merged `--eng-font` with `--font-path`
don't understand third question